modules/services/lorri.nix

```
@@ -34,7 +34,7 @@ in
           ProtectHome = "read-only";
           Restart = "on-failure";
           Environment =
-            let path = with pkgs; makeSearchPath "bin" [ nix gnutar gzip ];
+            let path = with pkgs; makeSearchPath "bin" [ nix git gnutar gzip ];
```

Perhaps
```suggestion
            let path = with pkgs; makeSearchPath "bin" [ nix gitMinimal gnutar gzip ];
```
is sufficient?

---

:+1: I didn't know `gitMinimal`, but it worked for me.